assets/js/article.js

```diff
@@ -1,7 +1,16 @@
 require('./common');
 // Temporary addition until comments are replaced
+
 if (window.commentsUseCoralTalk) {
 	require('o-comments-beta');
+
+	document.addEventListener('oComments.loginPrompt', () => {
+		const currentPath = new URL(location.href).pathname;
+		const commentsJumpAnchor = '#comments';
+
+		location.href = `https://accounts.ft.com/login?location=${encodeURIComponent(currentPath)}${encodeURIComponent(commentsJumpAnchor)}`;
```

Comment:
Very minor suggestion:

If we concatenate URL before and use encodeURIComponent only once for the concatenated URL it may be better for readability.